lib/utils.pm

```diff
@@ -57,6 +64,26 @@ sub wait_for_desktop {
     assert_screen 'generic-desktop';
 }
 
+# sle
```

Comment:
I think you are mixing up things. What you add is something like a completely new module to support initial setup of a JeOS image, not "SLE" per se. As mentioned in the related needles PR please just use the same base as equivalent on openSUSE, i.e. a "create_hdd_minimalx" job